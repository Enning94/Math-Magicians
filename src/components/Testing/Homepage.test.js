import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../Homepage';

it('Homepage component rendered correctly', () => {
  const tree = renderer
    .create(<HomePage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
