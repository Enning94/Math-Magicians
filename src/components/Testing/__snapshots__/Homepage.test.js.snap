// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component rendered correctly 1`] = `
<div
  className="home-page"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod optio non quidem nihil officia a error, magnam praesentium dolorum, ipsam suscipit. Voluptatem, iusto? Et commodi eaque aliquam, vero assumenda nam quod dolor nemo, facilis delectus autem magni quia temporibus voluptatum.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod optio non quidem nihil officia a error, magnam praesentium dolorum, ipsam suscipit. Voluptatem, iusto? Et commodi eaque aliquam, vero assumenda nam quod dolor nemo, facilis delectus autem magni quia temporibus voluptatum.
  </p>
</div>
`;
